Firefox Extension/views/instructions.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+    Welcome to CircuitVerse. Install the chrome extension + Google Slides Add On.
```

Comment:
Chrome ? @satu0king

Reply:
This was just template instructions page. Need to create a proper one.